frontend/components/commons/results/ResultsEmpty.vue

```diff
@@ -18,18 +18,28 @@
 <template>
   <div class="empty">
     <span class="empty__content">
-      <h1 class="empty__title">{{ title }}</h1>
+      <svgicon v-if="emptyIcon" :name="emptyIcon" width="50" height="50" color="#686a6d" />
+      <h1 class="empty__title">{{ emptyTitle }}</h1>
+      <p v-if="emptyDescription" class="empty__description" v-html="emptyDescription" />
     </span>
   </div>
 </template>
 
 <script>
 export default {
   props: {
-    title: {
+    emptyTitle: {
       type: String,
       default: "0 results found",
     },
+    emptyDescription: {
+      type: String,
+      default: undefined,
+    },
+    emptyIcon: {
+      type: String,
+      default: undefined,
+    },
```

Comment:
I think the naming is redundant since is called `ResultsEmpty`.

We can just parameterize as: `title`, `descrpition` and `icon`